_posts/en/newsletters/2020-09-02-newsletter.md

```diff
@@ -0,0 +1,252 @@
+  Signets are test networks where valid blocks must be
+  signed by one or more trusted keys; this allows them to be more
+  predictable for testing than fully decentralized test networks that
+  are often subject to vandalism.  [BIP325][] describes the signet
+  protocol, which anyone can use to create their own signet, but there
+  is a [desire][bitcoin core default signet] to include a default signet into
+  Bitcoin Core (and possibly other software) that
+  users will be able to enable with a single configuration option.
+  Folkson's first question is whether anyone has any objection to this.
+
+    If there is to be a default signet, Folkson posted several
+    additional questions about how it should be administered.  For
+    example, who should control its signing keys?  How many keys should
+    there be?  Under what circumstances should the default signet be
+    reset?  When should proposed consensus changes to mainnet Bitcoin be
+    implemented by the trusted signet miners/signers?
+
+    Anyone interested in helping answer these questions is encouraged to
+    respond.
+
+- **Standardizing temporarily trusted LN channels:** Roei Erez
+  [posted][temporarily trusted channels] to the Lightning-Dev mailing list a proposal to standardize
+  the way LN implementations deal with new channels when the
+  participants are willing to trust an unconfirmed deposit transaction.
+  For example, Alice pays Bob's established business to fund a
+  channel to her.  In that case, Alice may be willing to trust Bob to not
+  double spend his deposit transaction.  Bob can also safely accept
+  in-channel payments from Alice because he gets all his money back
+  anyway if the deposit transaction doesn't confirm.
+
+    Erez noted that several LN implementations already allow users to
+    opt-in to trusting certain unconfirmed channels but that the
+    different implementations vary in some aspects, such as what
+    `short_channel_id` to use until the deposit transaction confirms
+    (this value usually identifies the location of the deposit
+    transaction in the block chain, which doesn't apply to unconfirmed
+    transactions).
+
+    Participants in the mailing list discussion seemed supportive of the
+    proposal.
+
+## Recently transcribed talks and conversations
+
+*[Bitcoin Transcripts][] is the home for transcripts of technical
+Bitcoin presentations and discussions. In this monthly feature, we
+highlight a selection of the transcripts from the previous month.*
+
+- **nix-bitcoin**: nixbitcoindev appeared on the Stephan Livera Podcast to
+  discuss nix-bitcoin.  nix-bitcoin is an experimental but promising project
+  that aims to improve the security of installing Bitcoin and Lightning nodes. It
+  uses the functional Linux distribution NixOS where upgrading is equivalent to
+  installing from scratch due to atomicity. Through minimalism, reproducibility,
+  compartmentalization, and defense in depth, nix-bitcoin aims to reduce the attack
```

Comment:
It seems unclear what "defense in depth" actually means here. Perhaps omit and use "Through minimalismm, reproducibility, and compartmentalization," -- which is already a mouthful!

Reply:
Yeah I'd be happy with omitting "defense in depth".

It is referring to putting up multiple lines of defense isolating by users and isolating on the network level with systemd. Whereas compartmentalization is referring to the individual processes being separated. As a summary though I think omitting it makes sense.

Reply:
all good things come in three - ACK dropping `defense in depth`.